README.md

```diff
@@ -83,25 +92,29 @@ Search for and install `OpenFga.Sdk` in each of their respective package manager
 
 [Learn how to initialize your SDK](https://openfga.dev/docs/getting-started/setup-sdk-client)
 
-Without an API Token
+The documentation below refers to the `OpenFga.SdkClient`, to read the documentation for `OpenFga.SdkApi`, check out the [`v0.2.1` documentation](https://github.com/openfga/dotnet-sdk/tree/v0.2.1#readme).
+
+> The OpenFga.SdkClient will by default retry API requests up to 15 times on 429 and 5xx errors.
+
+#### No Credentials
 
 ```csharp
-using OpenFga.Sdk.Api;
 using OpenFga.Sdk.Client;
-using OpenFga.Sdk.Configuration;
+using OpenFga.Sdk.Client.Model;
 using OpenFga.Sdk.Model;
 
 namespace Example {
     public class Example {
         public static void Main() {
             try {
-                var configuration = new Configuration() {
+                var configuration = new ClientConfiguration() {
                     ApiScheme = Environment.GetEnvironmentVariable("OPENFGA_API_SCHEME"), // optional, defaults to "https"
                     ApiHost = Environment.GetEnvironmentVariable("OPENFGA_API_HOST"), // required, define without the scheme (e.g. api.fga.example instead of https://api.fga.example)
                     StoreId = Environment.GetEnvironmentVariable("OPENFGA_STORE_ID"), // not needed when calling `CreateStore` or `ListStores`
+                    AuthorizationModelId = Environment.GetEnvironmentVariable("OPENFGA_AUTHORIZATION_MODEL_ID"), // Optional, can be overridden per request
                 };
-                var openFgaApi = new OpenFgaApi(configuration);
-                var response = openFgaApi.ReadAuthorizationModels();
+                var fgaClient = new OpenFgaClient(configuration);
+                var response = fgaClient.ReadAuthorizationModels();
```

Comment:
```suggestion
                var response = await fgaClient.ReadAuthorizationModels();
```

and then you need to change `Main` to ` public static async Task Main(string[] args)`